.travis.yml

```diff
@@ -3,45 +3,48 @@ notifications:
     - oncall-dfusion@gnosis.io
   if: (branch = master) OR (tag IS present)
 if: (branch = master) OR (type = pull_request) OR (tag IS present)
+
+os: linux
+dist: xenial
```

Comment:
Just curious is there a reason to set these explicitly?

Reply:
Travis gives a warning about them not being specified and I agree with it that we should say which version we want instead of relying on a default.